README.md

```diff
@@ -1,4 +1,7 @@
-# Arg
+# arg
```

Comment:
Why the case change?

Reply:
Mostly for consistency. From what I've seen, most utility packages like this are kept as lowercase to indicate that it is a package, not a 'product'. Other Zeit packages like `ms`, `title`, `pkg`, and `ncc` refer to the package in the lower case form.

Reply:
Sounds good, though should ideally be in its own PR as it's an unrelated change.

Instead of doing that, can you put it into its own commit in this PR?